core/src/main/java/com/klaytn/caver/contract/ContractMethod.java

```diff
@@ -0,0 +1,315 @@
+package com.klaytn.caver.contract;
+
+import com.klaytn.caver.Caver;
+import com.klaytn.caver.abi.ABI;
+import com.klaytn.caver.methods.request.CallObject;
+import com.klaytn.caver.methods.response.Bytes;
+import com.klaytn.caver.methods.response.Bytes32;
+import com.klaytn.caver.methods.response.Quantity;
+import com.klaytn.caver.methods.response.TransactionReceipt;
+import com.klaytn.caver.transaction.response.PollingTransactionReceiptProcessor;
+import com.klaytn.caver.transaction.response.TransactionReceiptProcessor;
+import com.klaytn.caver.transaction.type.SmartContractExecution;
+import org.web3j.abi.datatypes.Type;
+import org.web3j.protocol.exceptions.TransactionException;
+import org.web3j.utils.Numeric;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Representing a Contract's method information.
+ */
+public class ContractMethod {
+
+    /**
+     * A caver instance.
+     */
+    Caver caver;
+
+    /**
+     * The input type. It always set "function".
+     */
+    String type;
+
+    /**
+     * The function name.
+     */
+    String name;
+
+    /**
+     * The list of ContractIOType contains to function parameter information.
+     */
+    List<ContractIOType> inputs;
+
+    /**
+     * The list of ContractIOType contains to function return value information.
+     */
+    List<ContractIOType> outputs;
+
+    /**
+     * The function signature string. ex) foo(uint256,uin256)
+     */
+    String signature;
+
+    /**
+     * The contract address.
+     */
+    String contractAddress;
+
+    /**
+     * Creates a ContractMethod instance.
+     */
+    public ContractMethod() {
+    }
+
+    /**
+     * Creates a ContractMethod instance.
+     * @param caver A Caver instance.
+     * @param type The input type. It always set "function"
+     * @param name The function name
+     * @param inputs The list of ContractIOType contains to function parameter information.
+     * @param outputs The list of ContractIOType
+     * @param signature The function signature string.
+     * @param contractAddress The contract address
+     */
+    public ContractMethod(Caver caver, String type, String name, List<ContractIOType> inputs, List<ContractIOType> outputs, String signature, String contractAddress) {
+        this.caver = caver;
+        this.type = type;
+        this.name = name;
+        this.inputs = inputs;
+        this.outputs = outputs;
+        this.signature = signature;
+        this.contractAddress = contractAddress;
+    }
+
+    /**
+     * Execute smart contract method in the EVM without sending any transaction.
+     * @param arguments A List of parameter that solidity wrapper type to call smart contract method.
+     * @param options An option to execute smart contract method.
+     * @return List
+     * @throws IOException
+     * @throws ClassNotFoundException
+     */
+    public List<Type> call(List<Type> arguments, SendOptions options) throws IOException, ClassNotFoundException {
```

Comment:
Can we support call or send with sendOptions parameter? (user can set default values of sendOptions, and those values will be used when sendOptions is not passed as a parameter)

Reply:
I think sendOption's from field is needed. 
Assuming that there is a function that only a smart contract owner or a sender with a specific role can view specific data, I think SendOptions' from field is necessary.